ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-commands.sh

```diff
@@ -37,15 +37,15 @@ else
     echo "OO_INSTALL_NAMESPACE is '$OO_INSTALL_NAMESPACE'"
 fi
 
-if [[ -n $NS_NAMESTANZA ]]; then
+if [[ -n "${NS_NAMESTANZA:-}" ]]; then
     OO_INSTALL_NAMESPACE=$(
         oc create -f - -o jsonpath='{.metadata.name}' <<EOF
 apiVersion: v1
 kind: Namespace
 metadata:
   $NS_NAMESTANZA
 EOF
-    )
+)
```

Comment:
This indentation change is incorrect, please remove

Reply:
ok, thanks!